public/app/features/alerting/components/PreviewQueryTab.tsx

```diff
@@ -13,14 +13,21 @@ export const PreviewQueryTab: FC<Props> = ({ data, height, width }) => {
   const [currentSeries, setSeries] = useState<number>(0);
   const theme = useTheme();
   const styles = getStyles(theme, height);
-  const series = useMemo(
-    () => data.series.map((frame, index) => ({ value: index, label: getFrameDisplayName(frame) })),
-    [data.series]
-  );
+  const series = useMemo<Array<SelectableValue<number>>>(() => {
+    if (data?.series) {
+      return data.series.map((frame, index) => ({ value: index, label: getFrameDisplayName(frame) }));
+    }
+
+    return [];
+  }, [data]);
```

Comment:
So data is either panelData or undefined, just making this clearer in the typings and here.